sklearn/exceptions.py

```diff
@@ -154,3 +155,12 @@ class UndefinedMetricWarning(UserWarning):
     .. versionchanged:: 0.18
        Moved from sklearn.base.
     """
+
+
+class SklearnDeprecationWarning(DeprecationWarning):
```

Comment:
Sorry for the noise, I just read the associated issue in more details where this kind of things were already discussed. I don't mind either way to be honest.